docs/section6/Rule6-10.md

```diff
@@ -0,0 +1,65 @@
+
+- For each building segment in building: `building_segment_b in B_RMR.building.building_segments:`  
+
+  - For each thermal block in building segment: `thermal_block_b in building_segment_b.thermal_blocks:`  
+
+    - For each zone in thermal block: `zone_b in thermal_block_b.zones:`  
+
+      - For each space in zone: `space_b in zone_b.spaces:`  
+
+        - Get interior lighting in space: `interior_lighting_b = space_b.interior_lighting`  
+
+          - Get interior lighting occupancy control type: `control_type_b = interior_lighting_b.occupancy_control_type`  
+
+        - Get normalized space lighting schedule: `normalized_schedule_b = normalize_space_lighting_schedule(space_b)`  
+
+        - Get matching space in P_RMR: `space_p = match_data_element(P_RMR, Spaces, space_b.id)`  
+
+          - Get normalized space lighting schedule: `normalized_schedule_p = normalize_space_lighting_schedule(space_p)`  
+
+        - Compare lighting schedule in B_RMR and P_RMR: `schedule_comparison_result = compare_schedules(normalized_schedule_b, normalized_schedule_p, building_open_schedule_b, 1)`  
+
+        - Get lighting space type: `lighting_space_type_b = space_b.lighting_space_type`  
+
+          - If lighting space types is employee lunch and break rooms, conference/meeting rooms, or classrooms (not including shop classrooms, laboratory classrooms, and preschool through 12th-grade classrooms), set occupancy sensor flag: `if lighting_space_type_b in ["LOUNGE/BREAKROOM", "CONFERENCE/MEETING/MULTIPURPOSE ROOM", "CLASSROOM/LECTURE HALL/TRAINING ROOM - PENITENTIARY", "CLASSROOM/LECTURE HALL/TRAINING ROOM - ALL OTHER"]: occ_sensor_flag = TRUE`  
+
+          **Rule Assertion:**  
+
+          - Case 1: If space requires occupancy sensor and interior lighting is modeled with full-auto on occupancy sensor control: `if ( occ_sensor_flag ) and ( control_type_b == "FULL-AUTO ON" ): PASS and return schedule_comparison_result`  
+
+          - Case 2: If space requires occupancy sensor and interior lighting is not modeled with full-auto on occupancy sensor control: `if ( occ_sensor_flag ) and ( control_type_b != "FULL-AUTO ON" ): FAIL`  
+
+          - Case 3: If space does not require occupancy sensor and interior lighting is modeled with full-auto on occupancy sensor control: `if ( NOT occ_sensor_flag ) and ( control_type_b == "FULL-AUTO ON" ): FAIL`  
+
+          - Case 4: If space does not require occupancy sensor and interior lighting is not modeled with full-auto on occupancy sensor control: `if ( NOT occ_sensor_flag ) and ( control_type_b != "FULL-AUTO ON" ): PASS and return schedule_comparison_result`  
+ 
+**Temporary Function note:**
+
+`compare_schedule_result = compare_schedules(lighting_schedule_p, normalized_schedule_b, building_open_schedule_p, reduction_factor_p)`
+
+(4 inputs, Schedule 1, Schedule 2, Mask Schedule, comparison factor)
```

Comment:
i think we need normalized schedule P and normalized schedule b. this would be the schedule normalized to the space level. I am not sure if we need the open schedule for this function. we could simply compare the normalized P schedule against the normalized b schedule. @mkarpman - any thoughts?

Reply:
@supriyagoel what you propose would work in this case because the two schedules must align 24/7. But for other building types the new function would be necessary - e.g., because for buildings over 5,000 SF we'd need space schedules to differ by the fractions prescribed in Table G3.7 and based on the specified controls 24/7 for building less than 5000SF, and during unoccupied hours only for larger buildings.